Resources/doc/getting-started.md

```diff
@@ -13,16 +13,16 @@ Alternatively, you can use [Composer](http://getcomposer.org) to install the sta
 Then, update your `app/config/AppKernel.php` file:
 
 ```php
-    public function registerBundles()
-    {
-        $bundles = [
-            // ...
-            new Dunglas\ApiBundle\DunglasApiBundle(),
-            // ...
-        ];
-
-        return $bundles;
-    }
+public function registerBundles()
+{
+    $bundles = [
+        // ...
+        new Dunglas\ApiBundle\DunglasApiBundle(),
+        // ...
+    ];
+
+    // 
```

Comment:
Just to keep the same syntax as [Symfony Bundle installation](http://symfony.com/doc/current/cookbook/bundles/installation.html). I guess that since it's there by default there is no need to specify it.